src/build.py

```diff
@@ -750,7 +751,9 @@ def CMakeCommandBase():
 
 def CMakeCommandNative(args, force_host_clang=True):
   command = CMakeCommandBase()
-  command.append('-DCMAKE_INSTALL_PREFIX=%s' % GetInstallDir())
+  command.extend(['-DCMAKE_INSTALL_PREFIX=%s' % GetInstallDir(),
+                  '-DCMAKE_SYSROOT=%s' %
+                  os.path.join(work_dirs.GetV8(), V8_LINUX_SYSROOT)])
```

Comment:
Is this function linux-only?

Reply:
Oops, it should only be used on Linux. fixed.